### DISCUSSION TITLE
Tags view in list separated by comma

### DISCUSSION THREAD

**Discussion by jekson** [laravel/nova-issues#6247] [Ideas & Feature Requests]
Laravel Version: 9.52.9
Nova Version: 4.25.1

Add on Index page view type without "View" button and show tags separated by comma in column
for example like in https://github.com/spatie/nova-tags-field

**Comment by crynobone** [comment]
This greatly impacts database queries performance due to having to query many-to-many on the Index. Marked as `blocked` for now.
